ui/imports/shared/panels/StatusAssetSelector.qml

```diff
@@ -165,5 +182,27 @@ Item {
                 }
             }
         }
+
+        Component.onCompleted: {
+            control.currentIndex = -1
+            control.popup.contentItem.header = searchBox
+        }
+
+        control.popup.onOpened: {
+            control.currentIndex = -1
```

Comment:
What's the purpose of this? Shouldn't the combobox retain its current item?

Reply:
so the thing is that the container in the combo box is StatusListView and it doesnt consider header as a selectable item, so it always start from the first item in the list instead of the header and hence this was needed